src/systems/filecoin_vm/interpreter/vm_registry.go

```diff
@@ -67,11 +56,11 @@ func _registerBuiltinActors() {
 
 	cron := &cronact.CronActorCode_I{}
 
-	RegisterActor(InitActorCodeID, &initact.InitActorCode_I{})
-	RegisterActor(CronActorCodeID, cron)
-	RegisterActor(AccountActorCodeID, &accact.AccountActorCode_I{})
-	RegisterActor(StoragePowerActorCodeID, &spowact.StoragePowerActorCode_I{})
-	RegisterActor(StorageMarketActorCodeID, &smarkact.StorageMarketActorCode_I{})
+	RegisterActor(builtin.InitActorCodeID, &initact.InitActorCode_I{})
+	RegisterActor(builtin.CronActorCodeID, cron)
+	RegisterActor(builtin.AccountActorCodeID, &accact.AccountActorCode_I{})
+	RegisterActor(builtin.StoragePowerActorCodeID, &spowact.StoragePowerActorCode_I{})
+	RegisterActor(builtin.StorageMarketActorCodeID, &smarkact.StorageMarketActorCode_I{})
```

Comment:
ditto

Reply:
There are many missing here, but I'm just trying to extract things, not fix them right now.